packages/app/src/components/TokenIconLabel.vue

```diff
@@ -17,12 +17,22 @@
         />
       </div>
     </AddressLink>
-    <div class="token-info" v-if="name && symbol">
-      <div class="token-symbol">
-        {{ symbol }}
-      </div>
-      <div class="token-name">
-        {{ name }}
+    <div class="token-info-container">
+      <div class="token-info" v-if="name && symbol">
+        <div class="token-symbol">
+          {{ symbol }}
+        </div>
+        <div class="token-name">
+          {{ name }}
+        </div>
+        <div style="display: flex; gap: 0.5rem">
```

Comment:
Would also suggest to display it in a separate column called `Type` or `Bridge`, would look a lot better as for me.

Reply:
Given that now we also have the _L1 token address_ column, I reverted back to badges because, with a separate column for the type, the table looks overwhelmed. Here is how it looks:
<img width="1265" alt="image" src="https://github.com/user-attachments/assets/766c4db5-38e0-4545-9934-808be0567c4d" />